examples/hand_pose_estimation/backend_keypoints.py

```diff
@@ -0,0 +1,1163 @@
+    y_lower_limits = np.less(uv_coordinates[:, 1], scoremap_size[1] - 1)
+    y_upper_limits = np.greater(uv_coordinates[:, 1], 0)
+    y_limits = np.logical_and(y_lower_limits, y_upper_limits)
+    keypoint_limits = np.logical_and(x_limits, y_limits)
+    return keypoint_limits
+
+
+def get_keypoints_mask(validity_mask, uv_coordinates, scoremap_size):
+    """ Extract Visibility mask for dominant hand.
+
+    # Arguments
+        validity_mask: Int value.
+        uv_coordinates: Numpy array of shape (21, 2).
+        scoremap_size: List of size (2).
+
+    # Returns
+        keypoint_limits: Numpy array of shape (21, 1).
+    """
+    validity_mask = np.squeeze(validity_mask)
+    keypoint_validity = np.greater(validity_mask, 0.5)
+    keypoint_limits = extract_keypoint2D_limits(uv_coordinates, scoremap_size)
+    keypooints_mask = np.logical_and(keypoint_validity, keypoint_limits)
+    return keypooints_mask
+
+
+def get_keypoint_limits(uv_coordinates, scoremap_size):
+    """ Extract X and Y limits.
+
+    # Arguments
+        uv_coordinates: Numpy array of shape (21, 2).
+        scoremap_size: List of size (2).
+
+    # Returns
+        X_limits: Numpy array of shape (21, 1).
+        Y_limits: Numpy array of shape (21, 1).
+    """
+    shape = uv_coordinates.shape
+
+    x_range = np.expand_dims(np.arange(scoremap_size[0]), 1)
+    x_coordinates = np.tile(x_range, [1, scoremap_size[1]])
+    x_coordinates.reshape((scoremap_size[0], scoremap_size[1]))
+    x_coordinates = np.expand_dims(x_coordinates, -1)
+    x_coordinates = np.tile(x_coordinates, [1, 1, shape[0]])
+    x_limits = x_coordinates - uv_coordinates[:, 0].astype('float64')
+
+    y_range = np.expand_dims(np.arange(scoremap_size[1]), 0)
+    y_coordinates = np.tile(y_range, [scoremap_size[0], 1])
+    y_coordinates.reshape((scoremap_size[0], scoremap_size[1]))
+    y_coordinates = np.expand_dims(y_coordinates, -1)
+    y_coordinates = np.tile(y_coordinates, [1, 1, shape[0]])
+    y_limits = y_coordinates - uv_coordinates[:, 1].astype('float64')
+
+    return x_limits, y_limits
+
+
+def create_multiple_gaussian_map(uv_coordinates, scoremap_size, sigma,
+                                 validity_mask):
+    """ Generate Gaussian maps based on keypoints in Image coordinates.
+
+    # Arguments
+        uv_coordinates: Numpy array of shape (21, 2).
+        scoremap_size: List of size (2).
+        sigma: Integer value.
+        validity_mask: Integer value.
+
+    # Returns
+        scoremap: Numpy array of shape (256, 256).
+    """
+    assert len(scoremap_size) == 2
+    keypoints_mask = get_keypoints_mask(
+        validity_mask, uv_coordinates, scoremap_size)
+    x_limits, y_limits = get_keypoint_limits(uv_coordinates, scoremap_size)
+    squared_distance = np.square(x_limits) + np.square(y_limits)
+    scoremap = np.exp(-squared_distance / np.square(sigma)) * keypoints_mask
+    return scoremap
+
+
+def keypoint_xy_coordinates(shape):
```

Comment:
function does not start with a verb does I don't know what is supposed to do inmediately.